app/pages/course-list-page/directives/course-status.js

```diff
@@ -0,0 +1,16 @@
+angular
+    .module('myApp')
+    .directive('courseStatus', function() {
+        return function(_, element, attrs) {
+            var courseDate = moment(attrs.courseStatus);
```

Comment:
??? you haven't to get variables from attributes.
See https://docs.angularjs.org/guide/directive & https://docs.angularjs.org/guide/component

Reply:
fix

Reply:
fix